fc/mef.py

```diff
@@ -736,230 +506,223 @@ def get_transform_fxn(data_beads, peaks_mef, mef_channels,
     if plot_filename is None:
         plot_filename = str(data_beads)
```

Comment:
As mentioned above, this seems like a dangerous misuse of the `__str__` function to populate a file name. Happens to work for current implementation, but I don't think the current `FCSData.__str__()` implementation faithfully performs what `__str__()` functions are expected to do (see discussion in #129 regarding `__repr__` and `__str__`).

I think at least referencing `FCSData.infile` would be better. Even that is ambiguous (file-like or string? maybe both work?). I'd be fine with just raising an error if `plot==True` but `plot_filename` isn't specified unless you can come up with a more reliable way of coming up with a filename from `data_beads`.